Servers/utils/iso42001.utils.ts

```
@@ -0,0 +1,620 @@
+  for (let clause of Clauses) {
+    for (let subClause of clause.subclauses) {
+      subClauses.push({
+        implementation_description: subClause.implementation_description || "",
+        auditor_feedback: subClause.auditor_feedback || "",
+      })
+    }
+  }
+  return subClauses;
+}
+
+const getDemoAnnexCategories = (): Object[] => {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)